### PR TITLE
Add supported car for 2019 Toyota Altis Hybrid

### PR DESCRIPTION
**Pull request by u8511049**
# Fingerprint
Toyota-Altis Hybrid-2019

# Car support
Toyota-Altis Hybrid-2019

https://my.comma.ai/5ceff72287a5c86c/1571534312818/1571537369543

Cabana
https://my.comma.ai/cabana/?route=5ceff72287a5c86c%7C2019-10-20--09-18-33&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F5ceff72287a5c86c%2F633c5011c172cc6c8f67084e2903ce1f_2019-10-20--09-18-33&seekTime=0

### CODE REVIEW

**Comment by u8511049** [comment]
update other links
https://my.comma.ai/5ceff72287a5c86c/1571453941876/1571457352486

https://my.comma.ai/cabana/?route=5ceff72287a5c86c%7C2019-10-19--10-59-02&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2F5ceff72287a5c86c%2F23574dab99ad8a81d33b9ee1e0b223e1_2019-10-19--10-59-02&seekTime=992

**Comment by rbiasini** [comment]
Thanks! My understanding is that the Altis is the same as the Corolla in the US market., correct? If yes, I would prefer to call it Corolla Hybrid (`COROLLAH_TSS2`) for consistency with the US market.

**Comment by rbiasini** [comment]
Also, fingerprint looks good. If you can change the name of the car, then it's good to be merged.

**Comment by u8511049** [comment]
Hi rbiasini,
corolla_tss2 and altish_tss2 have different dbc as below.
  CAR.COROLLA_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
  CAR.ALTISH_TSS2: dbc_dict('toyota_nodsu_hybrid_pt_generated', 'toyota_tss2_adas'),
I try to add fingerprint in corolla_tss2 but get an "Communication Issue between processes" error.
I think the different is about the gear. AltisH gear is "P R N D B"

**Comment by rbiasini** [comment]
Ah, good catch! Can we just call it Toyota Hybrid and add it to the Readme (see my diffs)? Then it's good to go.

**Comment by u8511049** [comment]
Sure. Thanks for help.
so, should I close this request?

**Comment by rbiasini** [comment]
Oh, why closing it? I mean, if you change the car name and add it to the readme I'll merge it in.

**Comment by u8511049** [comment]
Got it. :)
I had changed the car name and add it to the readme.
Thanks.

File: README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ Supported Cars
 | Toyota               | Rav4 Hybrid 2016           | TSS-P                | Yes     | Yes<sup>2</sup>| 0mph             | 0mph           | Toyota            |
 | Toyota               | Rav4 Hybrid 2017-18        | All                  | Yes     | Yes<sup>2</sup>| 0mph             | 0mph           | Toyota            |
 | Toyota               | Sienna 2018                | All                  | Yes     | Yes<sup>2</sup>| 0mph             | 0mph           | Toyota            |
+| Toyota               | Toyota Hybrid 2019         | All                  | Yes     | Yes            | 0mph             | 0mph           | Toyota            |
```
Comment (rbiasini) [inline README.md]: this should be:
```
| Toyota               | Toyota Corolla Hybrid 2020      | All                  | Yes     | Yes            | 0mph             | 0mph           | Toyota            |
```
And placed in alphabetical order in the table (after Corolla Hatchback)

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -15,6 +15,7 @@ class CAR:
   AVALON = "TOYOTA AVALON 2016"
   RAV4_TSS2 = "TOYOTA RAV4 2019"
   COROLLA_TSS2 = "TOYOTA COROLLA TSS2 2019"
+  TOYOTAH_TSS2 = "TOYOTA HYBRID TSS2 2019"
```
Comment (rbiasini) [inline selfdrive/car/toyota/values.py]: this should be 
```COROLLAH_TSS2 = "TOYOTA COROLLA HYBRID TSS2 2019"```

**Comment by rbiasini** [comment]
Almost. Sorry, I'm not being clear, my fault. See 2 comments in PR

**Comment by u8511049** [comment]
It's ok.
Please check it.
Thanks.

**Comment by rbiasini** [comment]
looks good!

**Pull request merged**